subprojects/packagefiles/nanobind/meson.build

```diff
@@ -0,0 +1,39 @@
+project(
+    'nanobind',
+    'cpp',
+    version: '2.0.0',
+    license: 'BSD-3-Clause',
+)
+
+py_mod = import('python')
+py = py_mod.find_installation()
+py_dep = py.dependency(embed: true)
+
+robin_map_dep = dependency('robin-map')
+
+incdir = include_directories('include')
+
+nanobind_lib = library(
+    'nanobind',
+    sources: [
+        'src/nb_internals.cpp',
+        'src/nb_func.cpp',
+        'src/nb_type.cpp',
+        'src/nb_enum.cpp',
+        'src/nb_ndarray.cpp',
+        'src/nb_static_property.cpp',
+        'src/common.cpp',
+        'src/error.cpp',
+        'src/trampoline.cpp',
+        'src/implicit.cpp',
+    ],
+    include_directories: [incdir],
+    install: true,
```

Comment:
nanobind's CMake files have an option whether to install it, do we also want that? If users can choose not to install it, we might want to use:

```meson
build_target(...,
    target_type: should_install ? 'library' : 'static_library',
)

Reply:
Nice catch. If I am reading the CMake configuration correctly it seems like it is only intended to be installed when not used as a subproject - so maybe I should just remove install altogether from this configuration?

Reply:
Entirely possible. In that case, also change library to static_library, I think...